src/rust/event-source/tests/test_crud_event_source.rs

```diff
@@ -2,13 +2,18 @@
 
 use std::time::SystemTime;
 
-use event_source::client::FromEnv;
+use clap::Parser;
 use grapl_utils::future_ext::GraplFutureExt;
 use rust_proto::graplinc::grapl::api::event_source::v1beta1 as es_api;
+use rust_proto_clients::{
+    get_grpc_client,
+    services::EventSourceClientConfig,
+};
 
 #[test_log::test(tokio::test)]
 async fn test_create_update_get() -> Result<(), Box<dyn std::error::Error>> {
-    let mut client = es_api::client::EventSourceServiceClient::from_env().await?;
```

Comment:
tldr these FromEnvs were scattershot everywhere, i've now made them standardized and `impl GrpcClientConfig`